Dockerfile

```diff
@@ -10,11 +10,15 @@ RUN mkdir /action
 WORKDIR /action
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends git openssh-client ca-certificates \
+  && apt-get install -y --no-install-recommends git openssh-client ca-certificates ruby-full \
   && apt-get purge -y --auto-remove \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && gem install bundler
 
 COPY package.json package-lock.json index.js /action/
 RUN npm ci --only=prod
 
+RUN useradd -ms /bin/bash deploy
+USER deploy
```

Comment:
this does nothing in GitHub actions as they are always run as root

Reply:
Without it bundler will complain about being run with root. But gem will still build, it really just ensures that Gem lock will have the latest version as the gem itself.
```
[8:17:38 PM] [semantic-release] [semantic-release-rubygem-with-pkg-root] › ℹ  Updating lock file with command `bundle install`
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and
installing your bundle as root will break this application for all non-root
users on this machine.
```